app/src/main/java/com/google/android/soundchecker/AudioEncoderDecoderActivity.kt

```diff
@@ -139,6 +139,8 @@ class AudioEncoderDecoderActivity : ComponentActivity() {
     private var mInputFileSampleRate = 0
     private var mInputFileStream: InputStream? = null
 
+    private var mOutputTrackCount = 0;
```

Comment:
Is this the number of active tracks? Or a historic count? Please add a comment.

Reply:
Added some comments.